src/main/java/seedu/eylah/expensesplitter/logic/commands/AddItemCommand.java

```diff
@@ -0,0 +1,49 @@
+package seedu.eylah.expensesplitter.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.eylah.commons.util.CollectionUtil.requireAllNonNull;
+
+import seedu.eylah.expensesplitter.logic.commands.exceptions.CommandException;
+import seedu.eylah.expensesplitter.model.Entry;
+import seedu.eylah.expensesplitter.model.Model;
+import seedu.eylah.expensesplitter.model.item.Item;
+
+
+/**
+ * Used to add items to the receipt.
+ */
+public class AddItemCommand extends Command {
+
+    public static final String COMMAND_WORD = "additem";
+
+    public static final String MESSAGE_SUCCESS = "The item %1$s has been added.";
+
+    private Entry toBeAdded;
+
+    /**
+     * Creates an AddItemCommand to add the specified {@code Item}
+     *
+     * @param item Item to be added.
+     * @param persons String array of persons to be added.
+     */
+    public AddItemCommand(Item item, String ... persons) {
+        requireAllNonNull(item, persons);
+        toBeAdded = new Entry(item, persons);
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        model.addEntry(toBeAdded);
+        return new CommandResult(String.format(MESSAGE_SUCCESS, toBeAdded));
+    }
```

Comment:
Is the Expense Splitter team not doing tagging for the number of people? Would be good to be able to differentiate between the item (which may be named with more than 1 word, like "green chilli" or "la zi ji") and the people. In that case you may want to explore the argument tokenizer and prefixes that has been used in Address Book previously.

Reply:
Hmm, what do you mean by tagging for the number of people?

Currently our main command is `additem -i la zi ji -p 5 -n bob -n calvin -n alan`

We doing -n on the PersonName, instead of the number of Person. Does this answer your question? Or do you feel that having the number of Person will make it simpler?

Reply:
@Jarrod-Bob Sorry, I've yet to implement that portion!